test_data/el-stability-check.yml

```diff
@@ -24,7 +24,7 @@ tasks:
 
 - name: run_task_matrix
   title: "Check block proposals from all client pairs"
-  timeout: 3m
+  timeout: 4m
```

Comment:
I saw a couple of assertoor checks failing due to the 3m timeout (less than 2-3%). Just rerunning the job solves the issue, just in case I raised the timeout, it may be cause by another issues and it still happens will be worth checking it out.